jboss/container/eap/galleon/config/mp/artifacts/opt/jboss/container/eap/galleon/definitions/fat-default-server/provisioning.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" ?>
+
+<installation xmlns="urn:jboss:galleon:provisioning:3.0">
+    <!-- Use the image version,  to avoid retrieving metadata.xml for un-released s2i FP-->
+    <feature-pack location="eap-s2i@maven(org.jboss.universe:s2i-universe)#23.0.0.Final">
```

Comment:
is the version correct?

Reply:
We used to use the upstream version. I agree that we could align with product version. This version is not documented, but still.